src/shared/components/matrix/myProfileForm/myProfileForm.component.tsx

```diff
@@ -90,7 +97,18 @@ export const MyProfileForm = () => {
           )}
         />
       </FieldContainer>
-      <FormActions withoutBackButton nextLabel="Save" isDisabled={!isDirty} isLoading={isSubmitting} />
+      <ButtonsContainer>
+        <FormActions withoutBackButton nextLabel="Save" isDisabled={!isDirty} isLoading={isSubmitting} />
+        <Button
+          size={ButtonSize.LARGE}
+          variant={ButtonVariant.PRIMARY}
+          type="button"
+          disabled={isSubmitting}
+          onClick={() => history.push(ROUTES.matrix)}
+        >
+          Go to skills matrix <ArrowRightIcon width={22} />
```

Comment:
Button has a property `icon` where you can pass `ButtonIcon.ARROW`, so there's no need in adding icon here

Reply:
Thanks, I didn't notice that. Changed ✅